metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -522,6 +583,11 @@ def generate_dataset_profile(  # noqa: C901 (complexity)
         row_count: int = profile.rowCount
 
         for column_spec in columns_profiling_queue:
+            if column_spec.type_ == ProfilerDataType.UNKNOWN:
+                logger.debug(
+                    f"profiling is disabled for {column_spec.column} as it's type is not supported for profiling"
+                )
+                continue
```

Comment:
Out of curiosity, what was the motivation for this change? Is there a reason we don't want to get nonnull counts or unique counts for columns with unknown types?

Reply:
Sorry, I should have commented about it. There are types where Profiling fails like GEOGRAPHY type in some dbs. 
I wanted to make sure we don't try to profile these type of columns as there column cardinality might break.